src/features/collapsible-sidebar/__tests__/CollapsibleSidebarLogo.test.js

```diff
@@ -32,4 +32,24 @@ describe('components/core/collapsible-sidebar/CollapsibleSidebar', () => {
 
         expect(sidebar).toMatchSnapshot();
     });
+
+    test('render logo element with linkUrl thats passed', () => {
+        const someUrl = '/some/url';
+        const sidebar = getWrapper({
+            linkUrl: someUrl,
+        });
+
+        const componentProp = sidebar.find('CollapsibleSidebarItem').prop('expandedElement');
+        expect(componentProp.props.children[1].props.href).toBe(someUrl);
```

Comment:
looking for the link based on specific index seems really brittle. Can you do another `find` for the `LinkBase` instead?